LeaderboardBackend/Models/Entities/NumericalMetric.cs

```diff
@@ -0,0 +1,96 @@
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace LeaderboardBackend.Models.Entities;
+
+/// <summary>
+///     Represents a `NumericalMetric` for a `Category`.
+/// </summary>
+public class NumericalMetric : BaseEntity
```

Comment:
These aren't so much numerical metrics as they are integer metrics.

Reply:
Was simply following the LucidChart. But yeah sure I'll rename it + add a migration.

Reply:
Okay so the migration won't be trivial, as I've since learned. In fact I think I might need to write custom SQL, which I don't fully know how, and will need quite a bit of time to learn. Maybe it would be worth at this point to start a new PR?

Reply:
There's no point in having to write a brand new migration with custom SQL just for this